week2/materials/homework/assignments.md

```diff
@@ -4,38 +4,38 @@
 
 Sync your changes to s3 and write down your s3 bucket url.
 
-bucket url: `?`
+bucket url: `https://s3.console.aws.amazon.com/s3/buckets/g3-week2-tithi?region=us-east-1&tab=objects`
 
 **Assignment 5:**
 
 Right now, the website does not support https. Explain which other AWS service needs to be integrated in order to achieve this.
 
-service name: `?`
+service name: `Though I see, the website already supports https. But I found one article(https://sectigostore.com/page/steps-to-install-an-ssl-certificate-on-aws-ec2-instance/), where I can follow the 5 steps to install SSL certificate to achieve this.`
```

Comment:
Indeed, that is a way to create certificates that can be used for something like https.However for S3 we can use it with a service that ends on _____front. 

Well done on the research though!